scrapy/downloadermiddlewares/ajaxcrawl.py

```diff
@@ -89,8 +88,7 @@ def _has_ajaxcrawlable_meta(text):
     if 'content' not in text:
         return False
 
-    text = _script_re.sub(u'', text)
-    text = _noscript_re.sub(u'', text)
-    text = html.remove_comments(html.replace_entities(text))
+    text = html.remove_tags_with_content(text, ('script', 'noscript'))
```

Comment:
Hmm, those templates without content all look a lot smaller than 32kb. So I just copied the beginning of such a template with the `<meta name="fragment" content="!"/>` in it and filled the rest of it (body) with lorem ipsum again.

Well that hurt. For comparability I kept the default loop number:
```
# master branch
1163.59052396

# w3lib (PR)
945.980225086
```
Since system load might have varied over that duration, another run with only 10.000 loops:
```
# master branch
11.6963450909

# w3lib (PR)
9.49189996719
```
I understand thats just a single page test-case. I don't feel like looking for more real-world cases though. Don't happen to have any broad crawl data to look up.